apprentice-react-o-shop/src/components/Auth/RegisterForm.js

```diff
@@ -0,0 +1,90 @@
+import React, {useState} from 'react';
+import {Link, useHistory} from 'react-router-dom';
+
+import InputField from '../InputField/InputField';
+import { formFields } from '../../utils/constants';
+
+function RegisterForm() {
+
+    const history = useHistory();
+
+    const [registerValues, setRegisterValues] = useState({
+        username: '',
+        password: '',
+        firstName: '',
+        lastName: '',
+    });
+
+    const [registerFormSubmitted, setRegisterFormSubmitted] = useState(false);
+
+    const { username, password, firstName, lastName } = registerValues;
+
+    const submitRegisterForm = (event) => {
+        event.preventDefault();
+        setRegisterFormSubmitted(true);
+        localStorage.setItem('user', username)
+        localStorage.setItem('password', password)
+        localStorage.setItem('firstName', firstName)
+        localStorage.setItem('lastName', lastName)
+        history.push('/home')
+    };
+
+    const updateRegisterValues = (event) => {
+        const {name, value} = event.target;
+        setRegisterValues((registerValues) => ({...registerValues, [name]: value}));
+    };
+
+    const inputFieldList = [];
+
+    for (const key in formFields){
```

Comment:
Rewrite this using a forEach method. You'll need to change formFields to an array.